.github/workflows/build_knapsack_system.yml

```diff
@@ -85,7 +85,7 @@ jobs:
           KNAPSACK_PRO_TEST_FILE_PATTERN: "spec/system/**/*_spec.rb" 
 
```

Comment:
I'm ignorant about Knapsack Pro, but I can see a comment above that says:
> if you use Knapsack Pro Queue Mode you must set below env variable...

Is this relevant to set up? If not, perhaps it's worth updating the comment to say why. Or perhaps it will be obvious once I read more..

Reply:
Yes, indeed - this seems relevant @dacook , thanks. As I understood it from that link, we might need it in case the build fails, so it is repeated on the same order. So far, I've only enabled queue mode for system specs - so I've set `KNAPSACK_PRO_FIXED_QUEUE_SPLIT: true`
now. It should not change the outcome of the build for now - let's see :-)